docs/install/containers/images/backend.md

```diff
@@ -304,6 +311,34 @@ And start a container.
 docker run -p 8080:8080 myproject:latest start
 ```
 
+### Adding `zope.conf` configuration fragments
+
+In the directory containing your `Dockerfile`, create a folder `etc/zope.conf.d`.
+Add your `zope.conf` configuration fragments there.
+
+Now add the following to your `Dockerfile`:
+
+```Dockerfile
+COPY /etc/zope.conf.d/*.conf /app/etc/zope.conf.d/
+```
+
+This ensures your fragments are deployed in the `zope.conf.d` folder, which then
+will be used to amend the `zope.conf` file prior to starting Plone.
+
+### Adding ZCML fragments
+
+In the directory containing your `Dockerfile`, create a folder `etc/package-includes`.
+Add your ZCML configuration fragments (named `*-meta.zcml`, `*-configure.zcml`,
+`*-overrides.zcml`) as files in that folder.
+
+Now add the following to your `Dockerfile`:
+
+```Dockerfile
+COPY /etc/package-includes/*.zcml /app/etc/package-includes/
```

Comment:
Where should this be placed in the Dockerfile?

Reply:
Good point.  Will fix.